tests/test_bpsconfig.py

```diff
@@ -105,7 +135,7 @@ class TestBpsConfigSearch(unittest.TestCase):
 
     def setUp(self):
         filename = os.path.join(TESTDIR, "data/config.yaml")
-        self.config = BpsConfig(filename, search_order=["baz", "bar", "foo"])
+        self.config = BpsConfig(filename, search_order=["baz", "bar", "foo"], defaults={})
```

Comment:
The `defaults` is an optional parameter so there should be no need for changing `BpsConfig` calls elsewhere. The default value for this parameters is `None` which instructs `BpsConfig.__init__()` to pre-populate the config with actual BPS defaults from `etc/bps_defaults.yaml` which previous version of the `BpsConfig.__init__()` was doing implicitly. Here, I just pass an empty dictionary to prevent `BpsConfig.__init__()` from doing this to simplify testing.